src/app/utils/common.util.ts

```diff
@@ -30,7 +30,7 @@ export class CommonUtil {
     return uniqid;
   }
 
-  static formatMemory(value: number | string): string {
+  static formatBytes(value: number | string): string {
```

Comment:
I only just noticed: we do not comply with the prefix definitions here: [](https://physics.nist.gov/cuu/Units/prefixes.html)
We use SI units but use a capital K. Filed a follow up jira to fix that throughout the web code